File: Dddml.Wms.HttpServices.PhpClient/Wms/Domain/LoginKey.php
<?php

namespace Wms\Domain;

use JMS\Serializer\Annotation\Type;
use Dddml\StringIdInterface;

class LoginKey
{
    /**
     * @Type("string")
     */
    private $loginProvider;

    /**
     * @return string
     */
    public function getLoginProvider()
    {
        return $this->loginProvider;
    }

    /**
     * @param string $loginProvider
     */
    public function setLoginProvider($loginProvider)
    {
        $this->loginProvider = $loginProvider;
    }

    /**
     * @Type("string")
     */
    private $providerKey;

    /**
     * @return string
     */
    public function getProviderKey()
    {
        return $this->providerKey;
    }

    /**
     * @param string $providerKey
     */
    public function setProviderKey($providerKey)
    {
        $this->providerKey = $providerKey;
    }


}

File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/LoginKey.java
package org.dddml.wms.domain;


public class LoginKey
{
    private String loginProvider;

    public String getLoginProvider()
    {
        return this.loginProvider;
    }

    public void setLoginProvider(String loginProvider)
    {
        this.loginProvider = loginProvider;
    }

    private String providerKey;

    public String getProviderKey()
    {
        return this.providerKey;
    }

    public void setProviderKey(String providerKey)
    {
        this.providerKey = providerKey;
    }


}

